src/smt/optimization_solver.cpp

```diff
@@ -28,6 +30,69 @@ using namespace cvc5::omt;
 namespace cvc5 {
 namespace smt {
 
+std::ostream& operator<<(std::ostream& out, const OptimizationResult& result)
```

Comment:
I wonder if this code should go to the smt2 printer rather than to be implemented here. @ajreynol what do you think?

Reply:
smt::Result also has its individual << operator.